dist/api_data.py

```diff
@@ -120,6 +120,14 @@ def __cmp__(self, other):
 
 # Per-file configuration
 file_config = format_meta + [
+    Config('access_pattern', 'default', r'''
+        It is recommended that workloads that consist primarily of
+        updates and/or point queries specify \c random.  Workloads that
+        do many cursor scans through large ranges of data specify
+        \c sequential and other workloads specify \c default.  The
+        option leads to an advisory call to an appropriate operating
+        system API where available''',
+        choices=['default', 'random', 'sequential']),
```

Comment:
AFAICT the man page states that `POSIX_FADV_NORMAL` is essentially a no-op.  It is what is used if no advice is given.  So I don't care much either way if we skip it or pass in normal.  The current code always calls it to be explicit and I kept that.